docs/UpdatedData.md

```diff
@@ -300,4 +300,18 @@ Leveraging the `deviceEvent` type and creating 2 new subTypes with the same stru
   "inputTime": "2020-05-12T08:40:00.000Z",
   "timezoneOffset": 60
 }
+{
+  "type": "deviceEvent",
+  "subType": "warmup",
+  "eventId": "Warm123456789",
+  "duration": { 
+    "value": 3,
+    "units": "hours"
+  },
+  "deviceId": "Id12345",
+  "deviceTime": "2018-02-01T00:00:00",
+  "time": "2020-05-12T08:50:08.000Z",
+  "inputTime": "2020-05-12T08:40:00.000Z",
+  "timezoneOffset": 60
```

Comment:
I have set a remark before on the previous PR, but ignored.
I think ignoring the timezoneOffset and set timezone instead is a better idea.
timezoneOffset  is useless.